test/Gridify.Tests/IssueTests/Issue293Tests.cs

```diff
@@ -0,0 +1,299 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Gridify.Tests.IssueTests;
+
+public class Issue293Tests
```

Comment:
You're convering more than cases than just the issue related ones, Please move this class out of the issues folder and name it `GridifyQueryTests` or `GridifyQueryValidationTests`